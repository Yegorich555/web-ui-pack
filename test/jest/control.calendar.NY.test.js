/**
 * @jest-environment ./test/jest.timeZoneEnv.js
 * @jest-environment-options {"timezone": "America/New_York"}
 */
import calendarTZtest from "./control.calendar.TZ";

/** Test calendar with timezone America/New_York */
describe(`control.calendar >> timezone: ${window.TZ}, offset: ${new Date().getTimezoneOffset()}`, () => {
  calendarTZtest();
});
