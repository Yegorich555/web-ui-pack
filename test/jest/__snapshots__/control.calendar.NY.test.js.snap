// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=false day picker firstOfWeek: Monday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Mo
        </li>
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
        <li>
          Su
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          28
        </li>
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
        <li
          next=""
          role="gridcell"
        >
          8
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=false day picker firstOfWeek: Sunday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Su
        </li>
        <li>
          Mo
        </li>
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          27
        </li>
        <li
          prev=""
          role="gridcell"
        >
          28
        </li>
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=false day picker firstOfWeek: Tuesday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
        <li>
          Su
        </li>
        <li>
          Mo
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
        <li
          next=""
          role="gridcell"
        >
          8
        </li>
        <li
          next=""
          role="gridcell"
        >
          9
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=true day picker firstOfWeek: Monday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Mo
        </li>
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
        <li>
          Su
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          28
        </li>
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
        <li
          next=""
          role="gridcell"
        >
          8
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=true day picker firstOfWeek: Sunday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Su
        </li>
        <li>
          Mo
        </li>
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          27
        </li>
        <li
          prev=""
          role="gridcell"
        >
          28
        </li>
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`control.calendar >> timezone: America/New_York, offset: 240 utc=true day picker firstOfWeek: Tuesday 1`] = `
<div
  calendar="day"
  id="txt2"
>
  <header>
    <button
      aria-hidden="true"
      tabindex="-1"
    >
      December 2022
    </button>
  </header>
  <div>
    <div>
      <ul
        aria-hidden="true"
      >
        <li>
          Tu
        </li>
        <li>
          We
        </li>
        <li>
          Th
        </li>
        <li>
          Fr
        </li>
        <li>
          Sa
        </li>
        <li>
          Su
        </li>
        <li>
          Mo
        </li>
      </ul>
      <ol
        role="grid"
        style="overflow: hidden;"
      >
        <li
          prev=""
          role="gridcell"
        >
          29
        </li>
        <li
          prev=""
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          1
        </li>
        <li
          role="gridcell"
        >
          2
        </li>
        <li
          role="gridcell"
        >
          3
        </li>
        <li
          role="gridcell"
        >
          4
        </li>
        <li
          aria-selected="true"
          role="gridcell"
        >
          5
        </li>
        <li
          role="gridcell"
        >
          6
        </li>
        <li
          role="gridcell"
        >
          7
        </li>
        <li
          role="gridcell"
        >
          8
        </li>
        <li
          role="gridcell"
        >
          9
        </li>
        <li
          role="gridcell"
        >
          10
        </li>
        <li
          role="gridcell"
        >
          11
        </li>
        <li
          role="gridcell"
        >
          12
        </li>
        <li
          role="gridcell"
        >
          13
        </li>
        <li
          role="gridcell"
        >
          14
        </li>
        <li
          role="gridcell"
        >
          15
        </li>
        <li
          aria-current="date"
          role="gridcell"
        >
          16
        </li>
        <li
          role="gridcell"
        >
          17
        </li>
        <li
          role="gridcell"
        >
          18
        </li>
        <li
          role="gridcell"
        >
          19
        </li>
        <li
          role="gridcell"
        >
          20
        </li>
        <li
          role="gridcell"
        >
          21
        </li>
        <li
          role="gridcell"
        >
          22
        </li>
        <li
          role="gridcell"
        >
          23
        </li>
        <li
          role="gridcell"
        >
          24
        </li>
        <li
          role="gridcell"
        >
          25
        </li>
        <li
          role="gridcell"
        >
          26
        </li>
        <li
          role="gridcell"
        >
          27
        </li>
        <li
          role="gridcell"
        >
          28
        </li>
        <li
          role="gridcell"
        >
          29
        </li>
        <li
          role="gridcell"
        >
          30
        </li>
        <li
          role="gridcell"
        >
          31
        </li>
        <li
          next=""
          role="gridcell"
        >
          1
        </li>
        <li
          next=""
          role="gridcell"
        >
          2
        </li>
        <li
          next=""
          role="gridcell"
        >
          3
        </li>
        <li
          next=""
          role="gridcell"
        >
          4
        </li>
        <li
          next=""
          role="gridcell"
        >
          5
        </li>
        <li
          next=""
          role="gridcell"
        >
          6
        </li>
        <li
          next=""
          role="gridcell"
        >
          7
        </li>
        <li
          next=""
          role="gridcell"
        >
          8
        </li>
        <li
          next=""
          role="gridcell"
        >
          9
        </li>
      </ol>
    </div>
  </div>
</div>
`;
